src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/utils.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice
+
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.authority.AuthorityUtils
+import org.springframework.security.oauth2.jwt.Jwt
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken
+
+fun createJwtAuthenticationToken(
```

Comment:
Could this file be renamed to `tokenHelper` or similar, please? "Utilities" usually erodes a lot.